swagger.yaml

```diff
@@ -2500,6 +2500,105 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /clusters/{cluster_id}/operators:
```

Comment:
My only concern is having `/operators` cluster endpoint and also `operators` cluster attribute. That may cause confusion to user, what to use and when. Not sure if your API must be backward compatible and not sure how you maintain it, but this may cause some wrong usage of the API.